client/src/entrypoints/admin/telepath/__snapshots__/widgets.test.js.snap

```diff
@@ -1,7 +1,5 @@
 // Jest Snapshot v1, https://goo.gl/fbAQLP
 
-exports[`telepath: wagtail.widgets.CheckboxInput it renders correctly 1`] = `"<input type="checkbox" name="sugar" id="id-sugar" checked="checked">"`;
```

Comment:
The new code sets the `checked` property on the element rather than adding/removing the attribute, so this change isn't reflected in the DOM / HTML representation. The corresponding test directly looks at the `checked` property anyhow.